app/views/contexts/email_context.py

```diff
@@ -0,0 +1,20 @@
+from flask import url_for
+
+
+def build_confirmation_email_context(email_confirmation_form):
```

Comment:
Minor, but in other places it's called `confirmation_email_form`, this should be changed to be consistent.